modules/mobile/app/services/mobile/v0/lighthouse_claims/proxy.rb

```diff
@@ -17,7 +17,7 @@ def claims_adapter
         end
 
         def claims_service
-          @claims_service ||= Mobile::V0::LighthouseClaims::Service.new(@user.icn)
+          @claims_service ||= BenefitsClaims::Service.new(@user.icn)
```

Comment:
Looking at the conversation in slack, it seems like the decision was to specify the client in the header. Will we be doing that later? So does this mean we didn't need to spend two months setting up our own keys? That's a bit obnoxious, but I'll agree that a header would be sufficient and reduce the likelihood that we (or someone else) will screw up the keys. 

I assume this means we don't need to subclass our services moving forward, right? Please LMK because Chika and I just worked on that yesterday and it sounds like we need to undo that work.

Reply:
yeah I figured we can handle that before this gets put into production but for now, QA is waiting to test this out and I've been debugging issues with our own rsa_key for the past few days so am happy to be able to not use the key. But yes, those two months were for nothing now :/ . 

Correct, we can just use the services as they are without injecting our own configuration.